src/Twig/MonitoringTwigExtension.php

```diff
@@ -76,7 +81,9 @@ public function embedMonitoring () : string
 
         return \sprintf(
             '<script src="%s"></script><script>window.TrackJS && TrackJS.install(%s)</script>',
-            $this->assetProvider->getUrl("js/trackjs.js"),
+            $assetUrl = null !== $this->assetHelper
```

Comment:
Can you please keep the (here) obsolete variable and generate the url before the `sprintf` call?